kura/examples/org.eclipse.kura.example.eddystone.scanner/src/main/java/org/eclipse/kura/example/eddystone/scanner/EddystoneScannerExample.java

```diff
@@ -0,0 +1,261 @@
+package org.eclipse.kura.example.eddystone.scanner;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import org.eclipse.kura.KuraException;
+import org.eclipse.kura.ble.eddystone.BluetoothLeEddystone;
+import org.eclipse.kura.ble.eddystone.BluetoothLeEddystoneCodec;
+import org.eclipse.kura.ble.eddystone.BluetoothLeEddystoneService;
+import org.eclipse.kura.bluetooth.le.BluetoothLeAdapter;
+import org.eclipse.kura.bluetooth.le.BluetoothLeService;
+import org.eclipse.kura.bluetooth.le.beacon.BluetoothLeBeaconListener;
+import org.eclipse.kura.bluetooth.le.beacon.BluetoothLeBeaconScanner;
+import org.eclipse.kura.cloud.CloudClient;
+import org.eclipse.kura.cloud.CloudService;
+import org.eclipse.kura.configuration.ConfigurableComponent;
+import org.eclipse.kura.message.KuraPayload;
+import org.osgi.service.component.ComponentContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EddystoneScannerExample implements ConfigurableComponent, BluetoothLeBeaconListener<BluetoothLeEddystone> {
+
+    private static final Logger logger = LoggerFactory.getLogger(EddystoneScannerExample.class);
+
+    private static final char[] HEX_CHAR_ARRAY = "0123456789ABCDEF".toCharArray();
```

Comment:
I agree with you about Options class if there is a lot of parameters (like Sensortag example), but here we have only 5.